### PR TITLE
RHDEVDOCS-4243 make some checklist updates for RHDEVDOCS-3972

### PR DESCRIPTION
**Pull request by rolfedh**
- Purpose: I ran the "OpenShift Docs Peer Review Checklist" but missed including a few improvements in the preceding PR, #47660. So, I've created this PR to handle those improvements.
- Aligned team: Dev Tools
- For branches: 4.11+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-4243 (originally [RHDEVDOCS-3972](https://issues.redhat.com//browse/RHDEVDOCS-3972))
- Direct link to doc previews: (Due to a technical issue, I cannot upload images to the share file server, so I must use this PDF instead.) [cicd_jenkins_openshift-jenkins.pdf](https://github.com/openshift/openshift-docs/files/9162833/cicd_jenkins_openshift-jenkins.pdf)
- SME review: @gabemontero @coreydaley @rgormley 
- QE review: @jitendar-singh 
- Peer review: @jc-berger 
- All reviews complete. Please merge now.

### CODE REVIEW

File: cicd/jenkins/openshift-jenkins.adoc [hunk 1]
```diff
@@ -6,9 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} 4.11 moves the `OpenShift Jenkins` and `OpenShift Agent Base` images to the `ocp-tools-4` repository at `registry.redhat.io` so that Red Hat can produce and update the images outside the {product-title} lifecycle. Previously, these images were in the {product-title} install payload and the `openshift4` repository at `registry.redhat.io`.
+{product-title} 4.11 moves the "OpenShift Jenkins" and "OpenShift Agent Base" images to the `ocp-tools-4` repository at `registry.redhat.io`. It also removes the "OpenShift Jenkins Maven" and "NodeJS Agent" images from its payload:
```
Comment (rolfedh) [inline cicd/jenkins/openshift-jenkins.adoc:9]: I added this abstract to meet the checklist requirement of providing a **single**-paragraph TLDR. Then I converted the two following paragraphs into bullets, which elaborate on the information in the abstract.

File: modules/relocation-of-openshift-jenkins-images.adoc [hunk 2]
```diff
@@ -1,8 +1,13 @@
+// Module included in the following assemblies:
+//
+// * cicd/jenkins/openshift-jenkins.adoc
 :_content-type: CONCEPT
 
 [id="relocation-of-openshift-jenkins-images_{context}"]
 = Relocation of OpenShift Jenkins images
 
+{product-title} 4.11 makes significant changes to the location and availability of specific OpenShift Jenkins images. Additionally, you can configure when and how to update these images.
```
Comment (rolfedh) [inline modules/relocation-of-openshift-jenkins-images.adoc:9]: I added this abstract to meet the checklist requirement of providing a single-paragraph TLDR.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**